addons/event_booth_sale/tests/common.py

```diff
@@ -21,6 +21,7 @@ def setUpClass(cls):
             'product_id': cls.event_booth_product.id,
         })
 
+        cls.env['account.tax.group'].search([]).country_id = cls.env.company.account_fiscal_country_id
```

Comment:
Same as above :)

Reply:
Put in the setup of `event` (`EventCase`) (note that this was already in a setup, unlike "above")